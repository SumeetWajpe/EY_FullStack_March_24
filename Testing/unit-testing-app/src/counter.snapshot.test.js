import renderer from "react-test-renderer";
import Counter from "./counter";

it("renders correctly", () => {
  const tree = renderer.create(<Counter />).toJSON();
  expect(tree).toMatchSnapshot();
});
