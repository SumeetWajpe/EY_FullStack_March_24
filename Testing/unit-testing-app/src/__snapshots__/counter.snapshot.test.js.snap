// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <p>
    10
  </p>,
  <button
    onClick={[Function]}
  >
    +++
  </button>,
]
`;
